src/cloudWatchLogs/commands/saveCurrentLogDataContent.ts

```diff
@@ -72,13 +74,14 @@ export async function saveCurrentLogStreamContent(
         vscode.window.showErrorMessage(
             localize(
                 'AWS.cwl.invalidEditor',
-                'Not a Cloudwatch Log stream: {0}',
+                'Not a Cloudwatch Log data source: {0}',
                 vscode.window.activeTextEditor?.document.fileName
             )
         )
     }
 
-    recordCloudwatchlogsDownloadStreamToFile({
+    recordCloudwatchlogsDownload({
```

Comment:
did the telemetry definition change? or are we using a different one?

Reply:
I changed it to a more general one because I don't believe it was correct in all cases. This code can be run on the result of a log group so we would be emitting the wrong telemetry in that case.

Reply:
Ok. That sounds like a functional change, which should be in a separate PR or commit. Don't want functional changes in "rename" commits.

Can leave it this time, but good practice to mention it in the commit message then.

Reply:
That makes sense. I'll try to keep this PR focused on renaming.